scripts/util.cjs

```diff
@@ -616,44 +653,50 @@ const generateCoingeckoIds = async () => {
     )
   }
   const assetPlatformsList = await assetPlatformsResponse.json()
-  const assetPlatformsMap = assetPlatformsList
-    .reduce((acc, platform) => {
-      // Manually add Solana chain identifier since it's not in the CoinGecko
-      // asset platforms list
-      if (platform.id === 'solana' && !platform.chain_identifier) {
-        platform.chain_identifier = 101
-      }
+  const assetPlatformsMap = assetPlatformsList.reduce((acc, platform) => {
+    // Manually add Solana chain identifier since it's not in the CoinGecko
+    // asset platforms list
+    if (platform.id === "solana" && !platform.chain_identifier) {
+      platform.chain_identifier = 101
+    }
 
-      acc[platform.id] = platform.chain_identifier
-      return acc
-    }, {})
+    acc[platform.id] = platform.chain_identifier
+    return acc
+  }, {})
 
-  return coinList.reduce((acc, coin) => {
-    Object.entries(coin.platforms)
-      .forEach(([platform, contractAddress]) => {
-        const chainId = assetPlatformsMap[platform]
-        if (!chainId || !contractAddress) {
-          return
-        }
+  const coingeckoIds = coinList.reduce((acc, coin) => {
```

Comment:
nit: maybe rename to `coingeckoIdsByChainId`, that way we don't assume this is a string array?